docs/release-notes/next-iris/FATE.md

```diff
@@ -0,0 +1,20 @@
+* Changed `Chain.block_hash` - in `VM_FATE_SOPHIA_2` it will return
+  `Some(<blockhash>)` for `Chain.block_height` (i.e. current generation)
+  previously it returned `None`. With Bitcoin-NG we do have the block hash of
+  the current generation, so no reason not to allow this.
+
+* Generalized accounts, allow access to the signed transaction within the authentication context:
+```
+switch(Auth.tx)
+  Spend(from, to, amount, payload) => ...
+  AENSTransfer(from, to, name) => ...
+  ...
+```
+  This enables more use-cases, for example in combination with PayingForTx.
+
+* Added more crypto primitives (mainly pairing operations) for BLS12-381. This
+  enables for example Zero-knowledge proofs and more multi-signature schemes.
+
+* Added functions related to strings. It introduces `to_list` and `from_list`
```

Comment:
Wasn't Char support altered somehow?

Reply:
Yes, there is also `Char.to_int` and `Char.from_int`

Reply:
Should be included then?